src/components/tech/JoinTeam.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button, Flex, Heading, Section, Text, Link } from "@radix-ui/themes";
+import Image from "next/image";
+import React from "react";
+
+export const JoinTeam = () => {
+  return (
+    <Section
+      mx={{ md: "94px", initial: "2" }}
+      className="bg-[#98BEC6] rounded-3xl"
+      px={{ md: "9", initial: "6" }}
+      py="6"
+    >
+      <Flex
```

Comment:
**suggestion (blocking):** Based on the design, it looks like the text takes up a little more than half the space available.  Can we try to use some flex properties to go for a 60 / 40 width for the text / image?

**suggestion (blocking):** Can we use the flex stretch properties to have the image fill the vertical space available, as per the design?  Could add some vertical margin to the image if it's not supposed to stretch all the way to the top / bottom of the text.